cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -259,5 +260,15 @@ func (c bundleCmd) generateMetadata(manifestsDir, outputDir string) error {
 	if err != nil {
 		return fmt.Errorf("error generating bundle metadata: %v", err)
 	}
+	if c.overwrite {
```

Comment:
If these files to not exist, labels/annotations should be written too. You can copy relevant code from `bundle create` here since that command will be deprecated and removed at some point soon.

Reply:
`generateFunc` would create these files and write them, and if `--overwrite` is true, these annotations would be rewritten. The files are being created before this if they do not exist.

Reply:
What happens if `--overwrite=false` but these files did not exist before `GenerateFunc` wrote them? These annotations/labels would not be written.

Reply:
Will modify this to write annotations when `--overwrite=false` and files weren't present before. And to confirm, we would not like the annotations being rewritten when the metadata is already present and overwrite is explicitly set to be `false`.

Reply:
Correct